blockprotocol/blockprotocol
# Display tax information when creating a subscription + other styling fixes

## 🌟 What is the purpose of this PR?




This PR adds tax information for when a user is creating a stripe subscription, and makes some other minor styling adjustments.

## 🔗 Related links





- [Asana task](https://app.asana.com/0/1203623179643287/1204060790341362/f) _(internal)_

## 🔍 What does this change?




See commits.

## 🛡 What tests cover this?



Manual testing.

## ❓ How to test this?



1. Checkout the branch and setup the local dev environment
2. Run the internal api locally on the `bw/automatic-tax` branch (if it hasn't merged yet)
3. Go the the `/settings/billing` page, and click on a subscription tier
4. Observe that UK tax has been applied to your total (this is the default tax region for the dev environment)

## 📹 Demo



<img width="577" alt="image" src="https://user-images.githubusercontent.com/42802102/221449970-f99c09e4-ca7c-4272-8dcf-501a219b5326.png">
